src/FakeItEasy/Core/AssemblyExtensions.cs

```diff
@@ -14,7 +14,16 @@ public static bool ReferencesFakeItEasy(this Assembly assembly)
         {
             Guard.AgainstNull(assembly, nameof(assembly));
 
+#if FEATURE_REFLECTION_GETASSEMBLIES
             return assembly.GetReferencedAssemblies().Any(r => r.FullName == TypeCatalogue.FakeItEasyAssembly.FullName);
+#else
+            var fakeItEasyLibraryName = TypeCatalogue.FakeItEasyAssembly.GetName().Name;
+            var context = Microsoft.Extensions.DependencyModel.DependencyContext.Default;
+
+            var runtimeLib = context.RuntimeLibraries
+                .SingleOrDefault(library => string.Equals(library.Name, assembly.Name(), System.StringComparison.Ordinal));

```

Comment:
Thanks, @jeremymeng. Don't take too much time on looking for docs (unless you can get them published! :-D). If the values you're using look good, that's enough for me for now. We'll hook up the tests and so long as things keep working, we'll muddle along.